src/main/java/it/pagopa/pm/gateway/utils/VPosRequestUtils.java

```diff
@@ -36,6 +36,8 @@ public class VPosRequestUtils {
     private static final String METHOD = "/method";
     @Value("${vpos.request.responseUrl}")
     private String vposResponseUrl;
+    @Value("${pgs.vpos.method.notifyUrl}")
+    private String methodNotifyUrl;
 
     @Autowired
     VPosUtils vPosUtils;
```

Comment:
Prefer constructor injection to field injection :)

Reply:
https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/202/commits/573a1269a57687a7bad8255ca25c8dfa03a01aae